backend/src/main/java/me/quizzl/backend/models/MultipleChoice.java

```diff
@@ -19,6 +19,16 @@ public MultipleChoice(String questionStr, String correctAnswer, String[] incorre
     public String[] getIncorrect() {
         return this.incorrectAnswers;
     }
+    public Boolean evaluateAnswer(Answer answer) {
```

Comment:
just a nit-pick: this could use an `@Override` annotation (if we even need to keep it at all)